webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -1274,6 +1275,22 @@ function createNodeTable(gridId, nope, contextPath, refresh) {
   $("#first_line_header input").addClass("form-control")
 
 
+  function resetColumns()  {
+    var table = $('#'+gridId).DataTable();
+    var data2 = table.rows().data();
+    table.destroy();
+    $('#'+gridId).empty();
+
+    delete params["ajax"];
+
+    dynColumns = Object.keys(allColumns).filter(function(c) { return !(defaultColumns.map(function(col) { return col.title}).includes(c))});
```

Comment:
marvelous how much boilerplate you need for a lambda `.map(_.title)` :)

Reply:
That's terrible ...

There is a MUCH better syntax that exists but i don't know if i can use it or not (some old browser may not have it) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions 

( x => x.title )